### PR TITLE
update instructions with BOSH CLI v2 usage

### PR DESCRIPTION
**Pull request by jcscottiii**
Came up when I was deleting someone's TOTP.

### CODE REVIEW

**Comment by jcscottiii** [comment]
one moment, let me update the expected output tables for the commands

**Comment by jcscottiii** [comment]
ready!

File: content/docs/ops/runbook/troubleshooting-bosh.md [hunk 1]
```diff
@@ -83,69 +83,51 @@ the Troubleshooting Bosh Managed VMS section below.
...
 ```sh
-$ bosh download manifest <DEPLOYMENT_NAME> ./<DEPLOYMENT_NAME>.yml
-$ bosh deployment ./<DEPLOYMENT_NAME>.yml
-$ bosh status
-
-# ... output shortened for brevity
-
-Deployment
-  Manifest: /path/to/<DEPLOYMENT_NAME>.yml
+$ bosh -d <DEPLOYMENT_NAME manifest > ./<DEPLOYMENT_NAME>.yml
```
Comment (jmcarp) [inline content/docs/ops/runbook/troubleshooting-bosh.md]: Should probably be `bosh -d <DEPLOYMENT_NAME> manifest > ./<DEPLOYMENT_NAME>.yml

File: content/docs/ops/runbook/troubleshooting-bosh.md [hunk 2]
```diff
@@ -83,69 +83,51 @@ the Troubleshooting Bosh Managed VMS section below.
...
-
-# ... output shortened for brevity
-
-Deployment
-  not set
-```
-
-You can get a list of all the current deployments by running the following
-command:
+In the pre-v2 BOSH CLI, you needed to set your deployment.
```
Comment (jmcarp) [inline content/docs/ops/runbook/troubleshooting-bosh.md]: We don't use the ruby bosh cli anymore. WDYT about dropping these notes and starting with `In order to see all the deployments...`?

File: content/docs/ops/runbook/troubleshooting-bosh.md (hunk 1 above)
Comment (jcscottiii) [reply content/docs/ops/runbook/troubleshooting-bosh.md]: whoops, i forgot the end `>`. will add.

File: content/docs/ops/runbook/troubleshooting-bosh.md (hunk 2 above)
Comment (jcscottiii) [reply content/docs/ops/runbook/troubleshooting-bosh.md]: will do!

**Comment by jcscottiii** [comment]
updated!

**Review by jmcarp — APPROVED** [review]

**Pull request merged**